README.md

```diff
@@ -4,7 +4,8 @@
 To be used in environment agnostic modules that need nextTick functionality.
 
 - When run in Node.js `process.nextTick` is used
-- In modern browsers microtask resolution is guaranteed by `MutationObserver`
+- In modern browsers, microtask resolution is guaranteed by `queueMicrotask`
```

Comment:
Let's rename _browsers_ to _engines_ (technically non-browser engines may also implement it, e.g. Node.js does, altough we stick to `process.nextTick` there)

Reply:
@medikoo Done!